Zilon.Core/Zilon.GlobeObserver/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Linq;
+
+using Microsoft.Extensions.DependencyInjection;
+
+using Zilon.Core.Persons;
+using Zilon.Core.World;
+
+namespace Zilon.GlobeObserver
+{
+    class Program
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6105716)